### PR TITLE
Get rid of cloud/cloud123 user

### LINKED ISSUE #58: Remove cloud test user (with password cloud123) from cloud-init
This was copied straight out of cloud-hypervisor's test scripts directory while getting started.

https://github.com/ubicloud/clover/blob/78f495904f4aef473d5dbb83165f8c5e09549364/rhizome/lib/vm_setup.rb#L157-L159

### PR DESCRIPTION
**Pull request by fdr**
This was very useful back when we'd routinely break networking and need to use serial teletype to figure out what was going on, but that development mode is so much more rare now that it's acceptable to require patching up the code in development to get access to the VM this way.

It was adapted directly from the cloud-hypervisor test scripts.

Fixes #58

### CODE REVIEW

**Comment by fdr** [comment]
Added in another commit that is related to the serial teletype being less useful than it once was.

File: rhizome/bin/runvm.rb [hunk 1]
```diff
@@ -1,12 +0,0 @@
...
-
-unless (vm_name = ARGV.shift)
-  puts "need vm name as argument"
-  exit 1
-end
-
-require_relative "../lib/common"
-require_relative "../lib/vm_setup"
-
-VmSetup.new(vm_name).exec_cloud_hypervisor
```
[the remark this answers is not in the document]
Comment (fdr) [reply rhizome/bin/runvm.rb:12]: oh hmmm I suppose I should

[the remark this answers is not in the document]
Comment (fdr) [reply rhizome/bin/runvm.rb:12]: alright, did that.

**Review by pykello — APPROVED** [review]
Looks good. I also tried it quickly & created a vm, and didn't see any problems.

**Pull request merged**